react/Table/Virtualized/index.jsx

```diff
@@ -28,10 +28,14 @@ const VirtualizedTable = forwardRef(
     },
     ref
   ) => {
-    const [order, setOrder] = useState('asc')
-    const [orderBy, setOrderBy] = useState(defaultOrder)
+    const [order, setOrder] = useState(
+      defaultOrder ?? { attribute: columns[0].id, order: 'asc' }
```

Comment:
using object in state is tricky but why not, here.

To avoid confusion I think we can have `orderBy` and `orderDirection` everywhere in all components... so here let's have `{ by: columns[0].id, direction: 'asc' }` here and let's reflect that everywhere

Other thing, we didn't have default value previously I think for old `orderBy` state (`defaultOrder` prop could have been `undefined`). Maybe we don't need `?? { attribute: columns[0].id, order: 'asc' }`